lib/configs/_base.js

```diff
@@ -4,6 +4,12 @@
  */
 "use strict"
 
+const { ESLint, version: ESLintVersion } = require("eslint")
+
+const version = ESLint.version || ESLintVersion
```

Comment:
Ah, ESLint v6 didn't have ESLint classes. We can use `Linter.version` instead.
https://eslint.org/docs/latest/developer-guide/nodejs-api#linterversionlinterversion